pkgs/top-level/all-packages.nix

```diff
@@ -17724,7 +17724,7 @@ with pkgs;
 
   hydraAntLogger = callPackage ../development/libraries/java/hydra-ant-logger { };
 
-  hydra-check = with python3.pkgs; toPythonApplication hydra-check;
+  hydra-check = with python310.pkgs; toPythonApplication hydra-check;
```

Comment:
If it still works on 3.9 we shouldn't bump this.

Reply:
i use `match case` in the program so 3.10 is required